AudioDeviceExample/ViewController.swift

```diff
@@ -250,9 +250,15 @@ class ViewController: UIViewController {
 
             let dimensions = previewView.videoDimensions
             var previewBounds = CGRect(origin: CGPoint.zero, size: CGSize(width: 160, height: 160))
-            previewBounds = AVMakeRect(aspectRatio: CGSize(width: CGFloat(dimensions.width),
-                                                           height: CGFloat(dimensions.height)),
-                                       insideRect: previewBounds)
+            if dimensions.width == 0 && dimensions.height == 0 {
+                previewBounds = AVMakeRect(aspectRatio: CGSize(width: CGFloat(160),
+                                                               height: CGFloat(120)),
+                                           insideRect: previewBounds)
+            } else {
+                previewBounds = AVMakeRect(aspectRatio: CGSize(width: CGFloat(dimensions.width),
```

Comment:
what happens if one of them is 0?

Reply:
it will crash too. but this is happening when the camera permission is denied and in that case `previewView` has a dimension 0, 0 so I think for now this works :)